automation/src/test/scala/org/broadinstitute/dsde/test/api/tagannotation/rawls/RawlsTestTaggedAs.scala

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.dsde.test.api.tagannotation.rawls
+
+import org.scalatest.Tag
+
+object MethodsTestObj1 extends Tag("org.broadinstitute.dsde.test.api.tagannotation.rawls.MethodsTest1")
```

Comment:
It'd be good to get rid of the `Obj1` suffix once testing is complete (if I'm understanding the upstream PR conversation correctly). Do you plan on doing that prior to merge?

Reply:
Yes. I have cleaned up and refactored the annotation tags. I managed to place all annotation tags in one file as well.